draft-packages/avatar/docs/Avatar.stories.tsx

```diff
@@ -40,121 +36,261 @@ DefaultStory.args = {
 
 DefaultStory.parameters = { controls: { exclude: ["isCompany"] } }
 
-export const DesignSheetDefault = () => (
-  <div
-    style={{
-      display: "grid",
-      gridTemplateColumns: "repeat(auto-fit, minmax(150px, max-content))",
-      gap: "2.5rem",
-    }}
-  >
-    {avatarStoryData["users"].map(story => (
-      <div>
-        <Heading tag="h2" variant="heading-2">
-          {story.title}
-        </Heading>
-        <br />
-        {(story.stories as AvatarProps[]).map((storyData: AvatarProps) => {
-          const avatarProps = storyData as AvatarProps
-          return (
-            <>
-              <Avatar {...avatarProps} />
-              <br />
-            </>
-          )
-        })}
-      </div>
-    ))}
-  </div>
+export const StickerSheetDefault = () => (
```

Comment:
Firstly, definitely an improvement on the example code by moving the data into the story, so good work!

I think we can clean up the duplication a little though (which means changes per column will be easier) by using some constants. Make sure the constants are still within the story so that it surfaces in the code sample for our consumers :)

```
export const StickerSheetDefault = () => {
  const PROPS_PHOTO_PERSONAL = {
    fullName: "Jane Doe",
    disableInitials: false,
    isCurrentUser: true,
    avatarSrc: "https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png",
  }

  const PROPS_INITIALS_PERSONAL = {
    fullName: "Jane Doe",
    disableInitials: false,
    isCurrentUser: true,
  }

  return (
    ...
    <Avatar {...PROPS_PHOTO_PERSONAL} size="xxlarge" />
    <Avatar {...PROPS_INITIALS_PERSONAL} size="xxlarge" />
    etc...
  )
}
```

The downside of course is that we have an object for the props as opposed to having them declared within the component tags _however_ we should have separate stories for proper usage anyway (I'm almost thinking perhaps we should disable the docs for sticker sheets, but we'll revisit that separately).

Reply:
Awesome I have done this! looks a lot cleaner